worker/scraper.py

```diff
@@ -105,6 +109,11 @@ def _store_items(feed):
         db.session.add(new_entry)
         db.session.commit()
 
+        new_item_count += 1
+
+    feed.item_count += new_item_count
+    db.session.commit()
+
```

Comment:
Merge in your scraper split first and put UserFeed at the end of feeds and UserCategory at the end of category